src/services/constants.js

```diff
@@ -9,8 +9,8 @@ module.exports = {
     "07": "July",
     "08": "August",
     "09": "September",
-    "10": "October",
-    "11": "November",
-    "12": "December"
-  }
-}
+    10: "October",
```

Comment:
Curious, what's the idea behind the inconsistency for months?

Reply:
Prettier thinks that looks right for some reason... I don't like that. Good eye.

I'll see if I can update the prettier config and push it.